frontend/src/components/query/query-builder/steps/order/order-case-tag.tsx

```diff
@@ -0,0 +1,70 @@
+"use client";
+import { OrderColumn } from "@/definitions";
+import { Icon, IconName, Tag, Text } from "@blueprintjs/core";
+import SquareIcon, { SquareIconSize } from "@/components/icon/square-icon";
+import { useTable } from "@/data/use-tables";
+
+interface OrderCaseTagProps {
+  orderCase: OrderColumn;
+  index: number;
+  removeable: boolean;
+  removeCase: (index: number) => void;
+}
+
+export default function OrderCaseTag({
+  orderCase,
+  index,
+  removeable,
+  removeCase,
+}: OrderCaseTagProps) {
+  const {
+    data: table,
+    isLoading: isLoadingTable,
+    error: tableError,
+  } = useTable(orderCase.column.table);
+
+  if (tableError && orderCase.column.table !== "aggregate") {
+    return null;
+  }
+
+  return (
+    <Tag
+      minimal
+      intent="none"
+      className={`${isLoadingTable ? "bp5-skeleton" : ""}`}
+    >
+      <div className="flex flex-row items-center gap-2 py-1">
+        <div className="flex flex-row items-center gap-1 flex-nowrap">
+          {table ? (
+            <SquareIcon
+              icon={table.icon as IconName}
+              color={table.color}
+              size={SquareIconSize.SMALL}
+            />
+          ) : (
+            <SquareIcon
+              icon="function"
+              color="gray"
+              size={SquareIconSize.SMALL}
+            />
+          )}
+          {table ? (
```

Comment:
I think I prefer && here instead of a ternary here for consistency, lets use the ternary only when the second option is not nullish

Reply:
Agreed. Good catch